app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -68,74 +62,92 @@ import org.mozilla.fenix.wallpapers.Wallpaper
 @Composable
 @Suppress("LongParameterList")
 fun WallpaperSettings(
-    wallpapers: List<Wallpaper>,
+    onLearnMoreClick: (String) -> Unit,
+    wallpaperGroups: Map<Wallpaper.Collection, List<Wallpaper>>,
```

Comment:
We will want doc strings for these, and I think we should move the `onLearnMoreClicked` down with the rest of the lambda parameters.